### DISCUSSION TITLE
Pressing E with the thumb‽

### DISCUSSION THREAD

**Discussion by giscus** [precondition/precondition.github.io#24] [Comments]
# Pressing E with the thumb‽

Exploring the history and reasoning behind a radical ergonomic keyboard layout and trying it out for myself.

https://precondition.github.io/pressing-e-with-the-thumb

**Comment by RacingMat** [comment]
great article! thanks
I'm definitely putting E on my scylla thumb cluster!

**Comment by mikeinliters** [comment]
This posts puts the "e" in "epic".

**Reply by mikeinliters** [reply]
*post

**Comment by sachamasry** [comment]
About a decade ago, the pain from typing became too frequent, so I started looking for revolutionary solutions. This led me to the Maltron keyboard, which seemed to be better in some layout areas (though not build quality) than the Kinesis of the time. As I was already changing to an alien keyboard, I opted for the layout change to Malt, having read their paper, with the 'E' key on the left thumb cluster.

The keyboard itself has been problematic, and is retired. However, I bought an Ergodox EZ, programming it to the Malt layout as best as possible, which I have now used at home for many years, successfully. For portable use, I have an Atreus, whose layout I have similarly ported over, again with the 'E' under the left thumb.

I have never been a particularly fast typist (maximum of ~75wpm), but I have found the 'E' key to be comfortable (and in a productive location) under the left thumb, and have never had recurring pain from either of the keyboards, despite them being flat and not caved like the Maltron or Kinesis. Even though I still have to use QWERTY keyboards (customers, laptop with no room for setting up the Atreus), I find the change to be instant and not a showstopper, but still much prefer the Malt layout overall, and attribute it mainly to the 'E' key.

Thanks for the well-researched article!

**Comment by matey-jack** [comment]
Thanks for writing this article! It's quite overdue to make this Thumb-E concept more popular! I am actually using a thumb-E mapping since 2014 and with standard "dumb" keyboards no less: https://github.com/matey-jack/gemuetliche-tastatur
My inspiration for this was mainly the idea of a "wide" letter mapping where the right hand's home position is one key farther out than what they teach in class. It's very little learning effort, since we can just move all the letters and punctuation keys with the hand position, only the rightmost (and less often used) spill over to the middle column).
I had seen a few other thumb-E layouts since then and had expected that this would become the standard for split keyboards with thumb cluster. After all, it's an enormous benefit for a very small learning effort! Recently I compared a few "easily learnable" layouts and found that they all have inferior metrics compared to a straight-forward letter mapping with E on a thumb key. https://matey-jack.github.io/key-layout-visualizer/
We need to tell more people that they can have all those benefits just by adding a bit of configuration to their computers! (It works on Linux, Windows, and Mac, although with different configuration techniques on each.)
